### PR TITLE
[TASK] Do not prefix class ComposerJson

### PR DESCRIPTION
**Pull request by sabbelasichon**
Resolves: rectorphp/rector#6405

### CODE REVIEW

File: utils/compiler/src/PhpScoper/StaticEasyPrefixer.php [hunk 1]
```diff
@@ -16,6 +16,8 @@ final class StaticEasyPrefixer
         'Doctrine\Inflector\Inflector',
         // for SmartFileInfo
         'Symplify\SmartFileSystem\SmartFileInfo',
+        // for ComposerJson because it is part of the public API. I.e. ComposerRectorInterface
+        'Symplify\ComposerJsonManipulator\ValueObject\ComposerJson'
```
Comment (TomasVotruba) [inline utils/compiler/src/PhpScoper/StaticEasyPrefixer.php:20]: This should be the only change.
Scoper is aliasing classes, so the 2 cases above should be delegated corretly.

Comment (sabbelasichon) [reply utils/compiler/src/PhpScoper/StaticEasyPrefixer.php:20]: Yes, was not sure about it. Because you did something for the SmartFileInfo.

**Comment by sabbelasichon** [comment]
The error seems unrelated to my change.

**Comment by TomasVotruba** [comment]
> The error seems unrelated to my change.

Yeah, I'm doing some releases with rector-phpstan-rules.

Let's give this a try :+1:

**Pull request merged**